UnitsNet/Comparison.cs

```diff
@@ -31,21 +31,21 @@ public sealed class Comparison
         ///     Absolute tolerance is when the absolute difference between the two values is not greater than the tolerance value.
         /// </summary>
         /// <param name="referenceValue">The first value. If using relative tolerance, it is the value which the relative tolerance will be calculated against.</param>
-        /// <param name="value">The value to compare to.</param>
+        /// <param name="otherValue">The value to compare to.</param>
         /// <param name="tolerance">The absolute or relative tolerance value.</param>
         /// <param name="comparisonType">Whether to use tolerance as an absolute or relative tolerance.</param>
         /// <returns></returns>
-        public static bool Equals(double referenceValue, double value, double tolerance, ComparisonType comparisonType)
+        public static bool Equals(double referenceValue, double otherValue, double tolerance, ComparisonType comparisonType)
```

Comment:
WRC target is a big pain in the ass, I've contemplated dropping the support for it, but we have a Microsoft package that depends on it which is kind of cool.

I have created an issue to generate a separate code base for WRC, to avoid all the #if defs, which I think will be an improvement since most people don't care about WRC anyway. Then hopefully one will meet less friction when adding new stuff, since adding extra code like operator overloads would by default not be included in WRC.

https://github.com/angularsen/UnitsNet/issues/409

Reply:
I did see that, and considered picking that up :)